src/parser/error_messages.txt

```diff
@@ -548,6 +545,7 @@ sigma X\ p X
 sigma X Y Z\ p X, q Y Z
 
 goal: LPAREN FLOAT COLON AFTER IO_COLON
+goal: LPAREN FLOAT COLON VDASH
```

Comment:
I think this error should become something like: illformed type annotatio n, here a few examples.